docs/content.md

```diff
@@ -0,0 +1,43 @@
+# What is Swift?
+
+Swift is a general-purpose programming language built using a modern approach to safety, performance, and software design patterns.
+
+The goal of the Swift project is to create the best available language for uses ranging from systems programming, to mobile and desktop apps, scaling up to cloud services. Most importantly, Swift is designed to make writing and maintaining correct programs easier for the developer. To achieve this goal, we believe that the most obvious way to write Swift code must also be:
+
+**Safe**. The most obvious way to write code should also behave in a safe manner. Undefined behavior is the enemy of safety, and developer mistakes should be caught before software is in production. Opting for safety sometimes means Swift will feel strict, but we believe that clarity saves time in the long run.
+
+**Fast**. Swift is intended as a replacement for C-based languages (C, C++, and Objective-C). As such, Swift must be comparable to those languages in performance for most tasks. Performance must also be predictable and consistent, not just fast in short bursts that require clean-up later. There are lots of languages with novel features — being fast is rare.
+
+**Expressive**. Swift benefits from decades of advancement in computer science to offer syntax that is a joy to use, with modern features developers expect. But Swift is never done. We will monitor language advancements and embrace what works, continually evolving to make Swift even better.
+
+> [wikipedia.org/wiki/Swift_(programming_language)](https://en.wikipedia.org/wiki/Swift_(programming_language))
+
+%%LOGO%%
+
+# How to use this image
+
+##### Pull the Docker Image From Docker Hub:
+
+```bash
+docker pull swift
+```
+
+##### Create a Container from the Image and Attach It:
+
+```bash
+docker run --privileged -i -t --name swiftfun swift:latest /bin/bash
```

Comment:
Docker won't accept us recommending --privileged. Let's add a note describing the problem. Hopefully someone else might fix it one day (short version: REPL depends on lldb which needs privileged kernel access or something)